docs/sdks/android/deeplink

```diff
@@ -0,0 +1,63 @@
+---
+id: extension
+title: Notification Extender Service
+---
+
+Deep Link Android
+--------------------
+
+1. Navigate to the app > AndroidManifest.xml and add the below code to it. As we are creating a deep link for our MainActivity.java file so we have to add this code in the MainActivity part. Below is the code which is to be added to the AndroidManifext.xml file. Comments are added in the code to get to know in more detail.
+
+2. 'AndroidManifest.xml'
+
+<intent-filter>
+  <action android:name="android.intent.action.VIEW" />
+  <category android:name="android.intent.category.DEFAULT" />
+  <category android:name="android.intent.category.BROWSABLE" />
+
+  <!--on below line we are specifying the host name and the scheme type in add your App name -->
+   <data
+        android:host="https://cleverpush.com"
+        android:scheme="cleverpush" />
+</intent-filter>
+  
+<!--below is the same filter as above just the scheme is changed to your app name -->
+<intent-filter>
+  <action android:name="android.intent.action.VIEW" />
+  <category android:name="android.intent.category.DEFAULT" />
+  <category android:name="android.intent.category.BROWSABLE" />
+  
+  <data
+    android:host="www.cleverpush.com"
```

Comment:
Customer needs to change host= right? so we should add comment here

Reply:
<!-- here the host must be customer's website -->
  <data
    android:host="www.cleverpush.com"
    android:scheme="cleverpush" />


Is it okay..??